org.salutatoi.Cagou.json

```diff
@@ -0,0 +1,1463 @@
+{
+    "app-id": "org.salutatoi.Cagou",
+    "default-branch": "0.7.0",
```

Comment:
It should be unset or `"branch": "stable"`.

Reply:
oh right, I'll change that thanks.